src/models/sftp-client.ts

```diff
@@ -27,7 +27,11 @@ export class SftpClient extends EventEmitter {
   public connect(config: IConfig) {
     return new Promise((resolve, reject) => {
       this._ssh = new SshClient();
-
+      
+      if(config.tryKeyboard){
+        this._ssh.once('keyboard-interactive', (name, instructions, instructionsLang, prompts, finish) => { finish([config.password]); })
```

Comment:
This line needs to be formatted, so code consistency is all the same across the project. I'll handle this.